### ISSUE TITLE
Replace goatsound

### ISSUE THREAD

**Issue by ETLegacyTeam** [etlegacy/etlegacy#497]
Author: @rmarquis (Spyhawk)
Date: 2014-01-28
Redmine Issue: 497, https://dev.etlegacy.com/issues/497

---

I understand it has a practical function (ie, warning people around that someone has been backstabbed by an enemy covops), but seriously, the goatsound sound should be replaced by something that fits the game better.

---

Related commit(s):
475044433f64ff9212fd5abf017e844047f33ccf - mod: MOD_BACKSTAB introduced refs 497 This limits..

**Comment by ETLegacyTeam** [comment]
From: @adanski (Dragonji)
Date: 2014-01-28T13:05:31Z

---

By the way, can it be completely disabled?

**Comment by ETLegacyTeam** [comment]
From: @rmarquis (Spyhawk)
Date: 2014-01-28T13:23:15Z

---

Dragonji wrote:
> By the way, can it be completely disabled?

There are a lot of thing that could improve the covops class, disabling the goatsound being one of them. But imho, this should be only optional, and needs to be discussed along other optional classes improvements (ie, after 2.71 is released and once we get some "official" future planning/vision for the mod from Radegast and others).

**Comment by ETLegacyTeam** [comment]
From: @rmarquis (Spyhawk)
Date: 2014-02-05T15:19:06Z

---

Just realized that the goat sound is *not* part of vanilla. It seems to originate from the OSP mod for RtCW, and has been reimplemented in most (if not all?) et mods, including etpro.  There's one occurrence in vanilla code, but it is commented out. The goat sound is also not broadcasted, only the attacker/victim can heard it.

Imho, we should completely get rid of it - I don't think that such "fun" feature makes sense in legacy mod.

**Comment by ETLegacyTeam** [comment]
From: @IR4T4 (IR4T4)
Date: 2014-02-05T16:37:39Z

---

Sounds create atmosphere ... in game or in maps. From my point of view I wouldn't remove a sound which is 'kind of standard' it ET. Ok - it isn't a vanilla sound but well known by players. I pretty much like this sound to frighten camping snipers ;)

**Comment by ETLegacyTeam** [comment]
From: @JanSimek (Radegast)
Date: 2014-02-06T02:23:57Z

---

Goat sound is the epitome of fun features, but I would make an exception for it and either keep it as it is or just make it optional. It really is pretty annoying when someone stabs you from behind and you hear goats. It makes you angry and wanting to get revenge.

**Comment by ETLegacyTeam** [comment]
From: @rmarquis (Spyhawk)
Date: 2014-02-06T16:44:09Z

---

> Sounds create atmosphere ... in game or in maps.

I full agree on that. However, I don't think the goatsound creates the "right" atmosphere for legacy mod, and I don't believe that "being standard" makes it good either.
I personally always have hated this sound (so I'm kinda biased here), because it creates a "WFT?" moment that simply doesn't fit (maybe except in fun oriented mod). Also, it isn't this sound that creates a revenge feeling - the backstab itself does.

> Goat sound is the epitome of fun features, but I would make an exception for it and either keep it as it is or just make it optional. 

To be honest, I think creating an optional cvar just for this sound is the road to bloatware. I'd rather bear it and heard this sound every time and keeping the mod core as simple and as clean as possible. But I'd still prefer to replace or remove it instead :)

**Comment by ETLegacyTeam** [comment]
From: @IR4T4 (IR4T4)
Date: 2014-02-06T18:09:02Z

---

Spyhawk wrote:
> ...because it creates a "WFT?" moment that simply doesn't fit ... 
hehe :D that's what I'm talking about ... it fits perfectly :P
Well it's kind of taste finally ... at least admins can decide to drop it by overwriting the sound with a blank wav.

**Comment by ETLegacyTeam** [comment]
From: @rmarquis (Spyhawk)
Date: 2014-02-06T18:37:35Z

---

> that’s what I’m talking about ...

Not really, let me rephrase it:

It's not "WTF? What just happened?" (ie, "backstab effect"), but "WTF is that shitty sound?", which is totally different. It also depends more on which atmosphere we want to give to legacy mod. The goat sound might perfectly fits NQ (and I'd agree here, it's in line with the mario goomba sound for example), but legacy...

**Comment by ETLegacyTeam** [comment]
From: @adanski (Dragonji)
Date: 2014-02-06T21:48:58Z

---

ETPro made it a "standard" so the argument it's fun sound is invalid IMO :P

**Comment by ETLegacyTeam** [comment]
From: @rmarquis (Spyhawk)
Date: 2014-02-06T22:25:36Z

---

Dragon> Having "Pro" in its name doesn't make a mod "pro" (but a good antilag, ettv feature and some anti-cheat counter measures help to achieve it) :P

**Comment by ETLegacyTeam** [comment]
From: @adanski (Dragonji)
Date: 2014-02-06T23:20:55Z

---

Being a ET standard in all the leagues makes it "pro" :P

**Comment by ETLegacyTeam** [comment]
From: @IR4T4 (IR4T4)
Date: 2016-08-10T15:09:53Z

---

The sound was played on any knife kill and not for the covert only!
 
This is actually pretty annoying :D The code has been changed to play the sound for backstab kills only. 

I can imagine a fleshy sound for this now.

**Comment by ETLegacyTeam** [comment]
From: @rmarquis (Spyhawk)
Date: 2019-01-10T11:39:49Z

---

Added a "meat punch" sound as attachment, which might be a good fit here. Found on the free "Youtube audio library":https://www.youtube.com/audiolibrary/soundeffects.

**Comment by mittermichal** [comment]
IMO the goat sound should stay. I think most of players associate it as part of game
